graphqlapi/src/main/java/io/stargate/graphql/web/resources/DmlResource.java

```diff
@@ -169,17 +164,29 @@ private GraphQL getGraphl(
       return null;
     }
 
-    GraphQL graphql =
-        graphqlCache.getDml(
-            keyspaceName,
-            (AuthenticationSubject) httpRequest.getAttribute(AuthenticationFilter.SUBJECT_KEY),
-            RequestToHeadersMapper.getAllHeaders(httpRequest));
-    if (graphql == null) {
+    try {
+      GraphQL graphql =
+          graphqlCache.getDml(
+              keyspaceName,
+              (AuthenticationSubject) httpRequest.getAttribute(AuthenticationFilter.SUBJECT_KEY),
+              RequestToHeadersMapper.getAllHeaders(httpRequest));
+      if (graphql == null) {
+        replyWithGraphqlError(
+            Status.NOT_FOUND, String.format("Unknown keyspace '%s'", keyspaceName), asyncResponse);
```

Comment:
Now that the resources are merged, and the term "keyspace" appears in a GraphQL-first context, I'm tempted to drop the whole "namespace" concept altogether. Users would create their keyspaces via the DDL service, and we can just use "keyspace" everywhere.

Reply:
Yes this is now very confusing imo.. Especially that the code switches back and forth between keyspace and namespace (like `SchemaSourceDao` uses namespace everywhere)..

Reply:
I'll remove namespace operations and unify naming in a follow-up PR.